src/app/portlets/dot-apps/dot-apps-export-dialog/dot-apps-export-dialog.component.spec.ts

```diff
@@ -0,0 +1,228 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { MockDotMessageService } from 'src/app/test/dot-message-service.mock';
+import { DotDialogModule } from '@components/dot-dialog/dot-dialog.module';
+import { CoreWebService } from 'dotcms-js';
+import { ReactiveFormsModule } from '@angular/forms';
+import { CommonModule } from '@angular/common';
+import { DebugElement } from '@angular/core';
+import { DotMessageService } from '@services/dot-message/dot-messages.service';
+import { InputTextModule } from 'primeng/inputtext';
+import { DotPipesModule } from '@pipes/dot-pipes.module';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { CoreWebServiceMock } from '@tests/core-web.service.mock';
+import { DotAppsExportDialogComponent } from './dot-apps-export-dialog.component';
+import { DotAutofocusModule } from 'projects/dot-rules/src/lib/directives/dot-autofocus/dot-autofocus.module';
+import { DotAppsService } from '@services/dot-apps/dot-apps.service';
+import { DotAppsExportConfiguration } from '@shared/models/dot-apps/dot-apps.model';
+import { By } from '@angular/platform-browser';
+
+export class DotAppsServiceMock {
+    exportConfiguration(_configuration: DotAppsExportConfiguration): Promise<string> {
+        return Promise.resolve('');
+    }
+}
+
+describe('DotAppsExportDialogComponent', () => {
+    let fixture: ComponentFixture<DotAppsExportDialogComponent>;
+    let comp: DotAppsExportDialogComponent;
+    let de: DebugElement;
+    let dotAppsService: DotAppsService;
+
+    const messageServiceMock = new MockDotMessageService({
+        'apps.confirmation.export.error': 'Error',
+        'dot.common.dialog.accept': 'Acept',
+        'dot.common.dialog.reject': 'Cancel',
+        'apps.confirmation.export.header': 'Export',
+        'apps.confirmation.export.password.label': 'Enter Password'
+    });
+
+    beforeEach(
+        waitForAsync(() => {
+            TestBed.configureTestingModule({
+                declarations: [DotAppsExportDialogComponent],
+                imports: [
+                    InputTextModule,
+                    DotAutofocusModule,
+                    DotDialogModule,
+                    CommonModule,
+                    ReactiveFormsModule,
+                    DotPipesModule,
+                    HttpClientTestingModule
+                ],
+                providers: [
+                    { provide: DotAppsService, useClass: DotAppsServiceMock },
+                    { provide: DotMessageService, useValue: messageServiceMock },
+                    { provide: CoreWebService, useClass: CoreWebServiceMock }
```

Comment:
If you are mocking `DotAppsService` and `DotMessageService` why do we need `CoreWebService`?